php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2139,25 +2140,23 @@ class NDB_BVL_Instrument extends NDB_Page
 
             $config = NDB_Config::singleton();
             $paths  = $config->getSetting('paths');
-            if (empty($subtest)) {
-                // check if the file/class exists
-                $BasePath = $paths['base'] . 'project/instruments/';
-                $TestName = $this->testName;
-                if (file_exists(
-                    $BasePath
-                    . "NDB_BVL_Instrument_$TestName.class.inc"
-                )
-                    || file_exists($BasePath . "$TestName.linst")
-                ) {
-                    // save possible changes from the control panel...
-                    $success = $controlPanel->save();
-                }
-
-                // display the control panel
-                $html = $controlPanel->display();
-                return $html;
 
+            // check if the file/class exists
```

Comment:
the `$subtest` variable was never defined, so `empty($subtest)` was always true, so I just removed the if statement.